.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -0,0 +1,18 @@
+<component name="InspectionProjectProfileManager">
```

Comment:
@shweta97 you seem to have removed one file without removing the others. Read and understand my comments carefully before committing.